@types/modules/pm/groups.d.ts

```diff
@@ -0,0 +1,20 @@
+declare interface PMGroupUpdateRequest {
+    prizeCategoryId: string;
+    prizeCatagoryName?: string;
+    prizeCategoryDescription?: string;
```

Comment:
In terms of attribute naming, can we go with "group" instead of "category", just so it's not confusing as to what objects are being edited? Also, we never include the id in an update request, as that would possibly change the id (which should be immutable). During a PATCH request, we can get the id from the URL, so we don't need it in the body.
```suggestion
    prizeGroupName?: string;
    prizeGroupDescription?: string;
```